docs/ballerina-by-example/examples/hello-world-service/hello-world-service.bal

```diff
@@ -6,7 +6,7 @@ endpoint http:ServiceEndpoint helloWorldEP {
 };
 
 @Description {value:"By default Ballerina assumes that the service is to be exposed via HTTP/1.1."}
-@http:ServiceConfig { basePath:"/helloWorld" }
+@http:ServiceConfig { basePath:"/hello" }
```

Comment:
Since the hello world sample has to be super simple, it is better to change the service name to hello and then remove the ServiceConfig annotation.